### ISSUE TITLE
RenderTexture memory leak despite destroying

### ISSUE THREAD

**Issue by Adam13531** [pixijs/pixi.js#5824]
### Expected Behavior

After destroying a RenderTexture, the memory that it used on the GPU is released.

### Current Behavior

The memory leaks.

### Steps to Reproduce

I wrote a small repro here: https://www.pixiplayground.com/#/edit/oByGkkG1sK149gGVaxyli

If you put a `return` at the top of `finishedLoadingAssets`, then no real Pixi objects will be created and the GPU memory usage for me is about 70 MB.

If, instead of adding a `return`, you comment out lines 31 and 32, then you'll also see the memory usage settle at about the same number.

However, with the code exactly as-is, the memory usage goes above 100 MB.

### Environment

- **`pixi.js` version**: 4.8.7 and 5.0.4 (it repros in both)
- **Browser & Version**: Chrome 74.0.3729.169
- **OS & Version**: Windows 10
- **Running Example**: https://www.pixiplayground.com/#/edit/oByGkkG1sK149gGVaxyli

### Investigation

I couldn't find any way to get the memory back down to 70 MB. I checked `PIXI.utils.TextureCache` and `PIXI.utils.BaseTextureCache`, but both were empty.

I'm also not totally sure why memory usage is so high to begin with for a RenderTexture. I'm seeing the same thing happen in my game where a single RenderTexture seems to add ~75 MB to GPU memory usage. The entire image itself is only ~200 KB on disk, and taking 75 MB causes lots of issues for low-end devices.

I think this issue is the same as #5747.

**Comment by ivanpopelyshev** [comment]
Framebuffer is disposed, stencil RenderBuffer is disposed too, Texture is disposed. What did we forget?

**Comment by ivanpopelyshev** [comment]
Also we have test like that: https://www.pixiplayground.com/#/edit/2SHfJOpn5hEnZjyWs9pHs , that fiddle crashes WebGL if stencilor something else isn't disposed. 

Maybe its the problem with memory manager itself,like, if memory goes about 100MB it goesn't want to be freed to system, it just stays that way.

**Comment by Adam13531** [comment]
@ivanpopelyshev 	
> Maybe its the problem with memory manager itself,like, if memory goes about 100MB it goesn't want to be freed to system, it just stays that way.

I think you may be right, since I added a configurable `numToLoad` property [at the top](https://www.pixiplayground.com/#/edit/ncU6KsH_F_VLeIF9zDVnD) and there doesn't seem to be any difference between `1` and `10`. Then again, maybe there's a problem in how I'm testing this (I'm relatively tired right now 😳).

Do you know why the amount of memory is increasing by something as large as 30-45 MB from such a small image? That's the "real" problem that I was investigating originally.

**Comment by Adam13531** [comment]
I investigated this some more, and I don't think there's a bug with Pixi.

Also, I tried to find out why I was using so much memory. There were two major findings:
- With antialiasing on, both the WebGL canvas itself and every texture uploaded to the GPU will end up using much more VRAM. However, antialiasing generally can't be replaced since you typically don't want jagged sprites.
- The number of bytes used per pixel by default is four thanks to a `format` of `gl.RGBA` and a `type` of `gl.UNSIGNED_BYTE`. I cut this down to two bytes per pixel by specifying a `type` of `gl.UNSIGNED_SHORT_4_4_4_4`. This involved patching Pixi since there's no way to overwrite the `type` property in both [V4](https://github.com/pixijs/pixi.js/blob/v4.x/src/core/renderers/webgl/TextureManager.js#L132) and [V5](https://github.com/pixijs/pixi.js/blob/ad02fe3c2ec4ed6c1fb6c659104ef99da68cc865/packages/core/src/textures/GLTexture.js).
